hexrd/material.py

```diff
@@ -108,7 +108,7 @@ class Material(object):
     the dmin parameter is used to figure out the maximum sampling for g-vectors
     this parameter is in angstroms
     '''
-    DFLT_DMIN = _angstroms(1.0)
+    DFLT_DMIN = _angstroms(0.5)
```

Comment:
Remember that in hexrdgui, we are currently initializing all of the materials' plane data at program start-up. This is tracked in hexrd/hexrdgui#497.

Reducing the default dmin to 0.5 might slow start-up time.